charts/federator/templates/configmap.yaml

```diff
@@ -43,7 +43,12 @@ data:
       # Filepath to one or more PEM-encoded server certificates to use as a trust
       # store when making grpc requests to remote backends
       {{- if $.Values.remoteCAContents }}
-      remoteCAStore: "/etc/wire/federator/ca/remote-ca.pem"
+      remoteCAStore: "/etc/wire/federator/conf/remote-ca.pem"
+      {{- end }}
+      useSystemCAStore: false
```

Comment:
```suggestion
```
That line exists (in the correct templated way) in line 53. hardcoding this to false doesn't seem like a good idea.